Update api documentation

forslund opened the pull request:
## Description

- Adds newer util functions along with the CommonPlaySkill class to the documentation on readthedocs.
- Replaces the sdkdoc option with the sphinxdoc so the same documentation can easily be generated locally

## How to test
Check the [generated documentation](https://mycroft-core.readthedocs.io/en/documentation-new-modules/)

## Contributor license agreement signed?
CLA [ Yes ]

pep8speaks commented:
Hello @forslund! Thanks for updating the PR.

 - There are no PEP8 issues in the file [`mycroft/skills/common_play_skill.py`](https://github.com/forslund/mycroft-core/blob/986e70ec9b6e2d436d6d7bbf37de666257386fd8/mycroft/skills/common_play_skill.py) !

 - In the file [`mycroft/util/__init__.py`](https://github.com/forslund/mycroft-core/blob/986e70ec9b6e2d436d6d7bbf37de666257386fd8/mycroft/util/__init__.py), following                are the PEP8 issues :
> [Line 280:13](https://github.com/forslund/mycroft-core/blob/986e70ec9b6e2d436d6d7bbf37de666257386fd8/mycroft/util/__init__.py#L280): [E722](https://duckduckgo.com/?q=pep8%20E722) do not use bare 'except'

forslund commented:
Thanks for spotting these things.
- typos fixed
- removed/replaced the outdated functions
- Added docstrings to the play_* functions
- added play_ogg to the documentation

KathyReid commented:
This documentation now makes me sing with joy :champagne: 
Merging ;-)
